apps/server/src/modules/tool/tool-launch/error/missing-licence.loggable-exception.ts

```diff
@@ -0,0 +1,35 @@
+import { ExternalToolMedium } from '@modules/tool/external-tool/domain';
+import { HttpStatus } from '@nestjs/common';
+import { BusinessError } from '@shared/common';
+import { EntityId } from '@shared/domain/types';
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+
+export class MissingLicenseLoggableException extends BusinessError implements Loggable {
+	constructor(
+		private readonly medium: ExternalToolMedium,
+		private readonly userId: EntityId,
+		private readonly contextExternalToolId?: string
+	) {
+		super(
+			{
+				type: 'MISSING_LICENSE',
+				title: 'Missing license',
+				defaultMessage: 'The user does not have the required license to launch this medium.',
```

Comment:
Either you have a general loggable for missing license like the class name and a general message or both specific